devices.js

```diff
@@ -376,6 +388,35 @@ const livolo = {
     },
 };
 
+const heiman = {
```

Comment:
Please inline this in the `configure()`, they are only used once so no need to put globally.